uniflow/op/extract/load/aws/s3_op.py

```diff
@@ -61,3 +61,66 @@ def __call__(self, nodes: Sequence[Node]) -> Sequence[Node]:
                 )
             )
         return output_nodes
+
+
+class ExtractS3HTMLOp(ExtractS3Op):
+    """Op to download a file from s3."""
+
+    # TODO: Refactor to merge ExtractS3HTMLOp and ExtractS3Op
+    # TODO: parse_html is a duplicate function
+
+    def parse_html(self, text):
```

Comment:
qq: should this be a private method to start with _.

Reply:
yeah if we want to keep it, this should be a private function.  We got `parse_html` from [here](https://github.com/CambioML/uniflow/blob/c5d617390b6f80550c0e4a53c033fd43c33f3c6e/uniflow/op/extract/load/html_op.py#L47), and we were trying to avoid just copying it over. If we end up having to copy it, we can definitely make it private.